config/database.yml

```diff
@@ -4,22 +4,36 @@
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
 #
+# MySQL version 2.x
+#   gem install mysql2
+#
+#   Ensure the MySQL 2 gem is defined in your Gemfile
+#   gem 'mysql2'
+
 default: &default
-  adapter: sqlite3
-  pool: 5
-  timeout: 5000
+  adapter: mysql2
+  encoding: utf8
+  database: predeploychecker
+  username: predeploychecker
+  password: <%= ENV['PDC_DATABASE_PASSWORD'] %>
+  port: 3306
 
 development:
   <<: *default
-  database: data/db/development.sqlite3
+  host: staging-predeploychecker.czrldetpmste.us-east-1.rds.amazonaws.com
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
+  adapter: sqlite3
+  pool: 5
+  timeout: 5000
   database: data/db/test.sqlite3
 
+# Currently there is no production PreDeployChecker, so setting the host to 127.0.0.1 for now
```

Comment:
The production Rails environment corresponds to a "deployed" state, not where the app is deployed.  When `RAILS_ENV=production`, web assets are expected to be compiled, the behavior of the autoloader changes, stack traces aren't presented to the user, etc.  The app is hardened and all the development tools are disabled.

We'll want to move the database `host:` from the `development` stanza above down here into the `production` section.  We should also avoid hardcoding the host and inject it as an environment variable instead.